include/wx/generic/grid.h

```diff
@@ -1481,6 +1481,22 @@ struct WXDLLIMPEXP_CORE wxGridSizesInfo
     wxUnsignedToIntHashMap m_customSizes;
 };
 
+// ----------------------------------------------------------------------------
+// wxGridArea
```

Comment:
Do we have to make this public? I.e. do we plan make `RefreshArea()` as part of public API? If yes, both the enum and the function should be documented. If not, could you please move the enum to `include/wx/generic/private/grid.h`?